index.html

```diff
@@ -23,15 +23,100 @@
     <section class="container" id="headline">
       <h1 class="head1" id="title">I’m Mariane Glad to see you!</h1>
       <p class="description" id="title-desc">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-      <h2 class="head2" id="headline2">LET’S CONNECT</h2>
+      <h3 class="head3" id="headline2">LET’S CONNECT</h3>
       <ul class="connect-lists">
         <li class="connect-items" id="connect-item1"><a href="https://twitter.com/RanjanAngom"><img src="images/twitter.png" alt="twitter link"></a></li>
         <li class="connect-items" id="connect-item2"><a href="https://linkedin.com/in/angom-chittaranjan"><img src="images/linkedin.png" alt="linkedin link"></a></li>
         <li class="connect-items" id="connect-item3"><a href="https://medium.com/"><img src="images/medium.png" alt="medium link"></a></li>
         <li class="connect-items" id="connect-item4"><a href="https://github.com/AngomRanjan"><img src="images/github.png" alt="github link"></a></li>
         <li class="connect-items" id="connect-item5"><a href="https://angel.co/"><img src="images/angellist.png" alt="angellist link"></a></li>
       </ul>
-    </section>
+    </section>    
   </div>
+  <section class="grid-container" id="works">
+    <div class="cards" id="card-1">
+      <div class="snapshoot-placeholders" id="snapshoot-placeholder1"><img src="images/img-project1.png" alt="Project-1 Snapshoot"></div>
+      <div class="info-blocks" id="info-block1">
+        <h1 class="project-titles" id="project-title1">Tonic</h1>
+        <h2 class="project-infos" id="info-project1">CANOPY <span class="gray">&bull; Back End Dev &bull; 2015</span></h2>
+        <p class="project-descriptions" id="desc-project1">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="tag-lists">
+          <li class="tags">html</li>
+          <li class="tags">css</li>
+          <li class="tags">javascripts</li>
+        </ul>
+        <button class="btn" type="button">See Project</button>
+      </div>
+    </div>
+    <div class="cards" id="card-2">
+      <div class="snapshoot-placeholders" id="snapshoot-placeholder2"><img src="images/img-project2.png" alt="Project-2 Snapshoot"></div>
+      <div class="info-blocks" id="info-block2">
+        <h1 class="project-titles" id="project-title2">Multi-Post Stories</h1>
```

Comment:
- [ ] `h1` is only used once for the title of the page, so kindly use another `hx` tag here.